cmd/vic-machine/create/create.go

```diff
@@ -861,6 +865,17 @@ func (c *Create) generateCertificates(ca bool) ([]byte, *certificate.KeyPair, er
 		log.Warnf("Failed to stash client certificate for later application level validation: %s", err)
 	}
 
+	// if openssl is present, try to generate a browser friendly pfx file
```

Comment:
What is browser friendly cert? Can we add a comment explaining what's going on here?